CHANGELOG.md

```diff
@@ -1,5 +1,188 @@
 # Changelog
 
+## 🔎 1.1.0
+
+Wrangler 1.1.0 includes a number of improvements to documentation and project stability, including: 
+
+- ### Features
+
+  - **Add message module for terminal output - [ashleymical], [issue/219] [pull/263]**
+
+    We've made improvements to Wrangler's terminal output functionality, with support for various log levels and implementations in Wrangler's API for easily using the log levels in future development.
+
+    [ashleymical]: https://github.com/ashleymichal
+    [issue/219]: https://github.com/cloudflare/wrangler/issues/219
+    [pull/263]: https://github.com/cloudflare/wrangler/pull/263
+
+  - **Adds more descriptive subdomain errors - [EverlastingBugstopper], [issue/207] [pull/259]**
+
+    Warns users when attempting to add a subdomain w/o account_id in wrangler.toml
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/207]: https://github.com/cloudflare/wrangler/issue/207
+    [pull/259]: https://github.com/cloudflare/wrangler/pull/259
+
+  - **Allow custom webpack configuration in wrangler.toml - [EverlastingBugstopper], [issue/246] [pull/253]**
+
+    Users can now specify custom webpack configuration filename, via a `webpack_config` key in `wrangler.toml`.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/246]: https://github.com/cloudflare/wrangler/issue/246
+    [pull/253]: https://github.com/cloudflare/wrangler/pull/253
+
+  - **Enforce one webpack config - [xtuc], [pull/245]**
+
+    We don't support multiple webpack configurations, for better UX we should make sure to enforce it.
```

Comment:
i'd elaborate on this more- what happened before? how was this going . wrong? what does the . error message look like?

Reply:
This error occurred when users exported more than one configuration in their webpack.config.js. The new error message we have gives users that information rather than a cryptic error message

Reply:
@EverlastingBugstopper the example test that @xtuc included in this PR was specifically around webpack `entrypoints` - I indicated that in the updated version of this commit, but can you confirm that I understood this correctly?

Reply:
This PR is meant to provide users w/a more helpful error message when their webpack.config.js looks like this: https://github.com/jrf0110/cf-workers-boilerplate/blob/master/webpack.config.js

It has to do with more than one export in their webpack.config.js. This is also why I introduced a separate PR allowing users to specify a custom filename for their webpack configuration